### ISSUE TITLE
Держать в README актуальный список необходимых пакетов

### ISSUE THREAD

**Issue by jrogov** [mbuzdalov/latex-styles#31]
Было бы удобно сразу знать, какие понадобятся и скачивать их сразу пачкой.

Бонусом можно написать готовую команду для tlmgr (который, вроде как, единственный пакетный менеджер и стандарт де-факто..?) для простой установки вида
```shell
tlmgr install [список пакетов]
```

**Comment by mbuzdalov** [comment]
Попробую сделать, хотя tlmgr по нынешним временам работает далеко не всегда, особенно в дистрибутивах линукса, где эта работа, как правило, переложена на соответствующие менеджеры пакетов.

Видимо, сделать надо будет так, чтобы и для популярных линуксовых менеджеров пакетов соответствующий список был. Хотя наиболее правильным вариантом было бы наконец опакетить этот класс и проставить в нем зависимости. Но не знаю, взлетит ли вообще этот наиболее правильный вариант :)

**Comment by jrogov** [comment]
@mbuzdalov Да, конечно в идеале было бы опакетить, но деталей и подводных камней этого процесса, увы, не знаю.
Как простую и быструю меру предлагаю содержать список названий используемых пакетов, как их можно найти на CTAN.
